topsort-endpoints.yml

```diff
@@ -103,9 +103,38 @@ components:
 
   schemas:
     AuctionRequest:
-      required:
+      type: object
+      anyOf:
```

Comment:
```suggestion
      oneOf:
```

Reply:
oneOf will no longer allow running sponsored listings and banner ads in a single request. But I see your point, it should either be the homepage thing or the anyOf for the other two.

Reply:
You are right, and actually that brings another question. The current way it's defined allows to run a banner campaign but with the wrong slot type.

For now maybe we should just leave the schema as it was, and in a v2 we develop a better model, maybe having different endpoints for banners and sponsored listings.

Reply:
Guys can we move this forward? What else is needed to push? We need to start giving the info to Airlift and updating this before they start working on the wrong documentation.

Reply:
@sk- I just pushed a change to define Slots according to the listings type.